mindmeld/converter/dialogflow.py

```diff
@@ -385,31 +360,74 @@ def create_mindmeld_init(self):
                             "Please check if your intent file"
                             "names are correctly labeled."
                         )
+                        return
 
                     datastore = json.load(source)
-                    replies = []
-
                     for response in datastore["responses"]:
-                        for message in response["messages"]:
-                            language = message["lang"]
-
-                            if "speech" in message:
-                                data = message["speech"]
-                                replies = data if isinstance(data, list) else [data]
-                                function_name = "renameMe" + str(i) + "_" + language
-                                handles = [
-                                    "intent='%s_%s'"
-                                    % (self.clean_name(datastore["name"]), language)
-                                ]
-
-                                target.write(
-                                    "\n\n\n"
-                                    + self.create_function(
-                                        handles=handles,
-                                        function_name=function_name,
-                                        replies=replies,
+                        message = response["messages"][0]
```

Comment:
consider refactoring this code into a separate function/class? MessageConverter?

Reply:
done